docs/sorting-algorithms/heap-sort.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: Heap Sort
+slug: /sorting-algorithms/heap-sort
+---
+
+# Introduction
+
+Heap Sort is a sorting algorithm which extracts the maximum or minimum element from the array and swap it with the last element of the array 
+for sorting in ascending or descending order N times.It is similar to Selection Sort, the only difference is that it uses Heap data structure 
```

Comment:
`It is similar to the Selection Sort, with the only difference...`

Reply:
done